### ISSUE TITLE
No values displayed.

### ISSUE THREAD

**Issue by marmitol** [ebaauw/homebridge-ups#5]
Hello.
I have installed and it does not show the values of:
Eaton 3S 850 Battery: set **Voltage from nullV to nullV**
Eaton 3S 850: set Current Consumption from null W to null W
Eaton 3S 850 Battery: set Voltage from nullV to nullV

![image](https://user-images.githubusercontent.com/61157812/196051697-88604d79-48a1-4d06-b084-2b7c4631768f.png)
![image](https://user-images.githubusercontent.com/61157812/196051907-ac761a2d-b2ab-4e43-8522-e2bcbcbe99ba.png)
![image](https://user-images.githubusercontent.com/61157812/196051604-0585be56-b263-4083-9939-6cac7f79ed74.png)

Thank you very much

**Comment by ebaauw** [comment]
> Eaton 3s 850: set Voltage to nullV

Homebridge UPS reads the mains voltage from `input.voltage`, which your UPS doesn't report.  I suppose I could use `output.voltage` instead, or simply not expose _Voltage_.

> Eaton 3S 850 Battery: set Voltage from nullV to nullV

Homebridge UPS reads the battery voltage from `battery.voltage`, which your UPS doesn't report.  I'm afraid the only solution here, is not to expose the battery _Voltage_.

> set Current Consumption to nullW

Homebridge UPS computes the consumption from `ups.load` (which indicates the load as percentage) and `ups.realpower.nominal`, which your UPS doesn't report.  I suppose I could use `ups.power.nominal` instead.

**Comment by ebaauw** [comment]
Could you try beta v1.0.3-0?  This version has a more robust parsing of UPS state, and should update power consumption and mains voltage for your UPS.  Battery voltage will remain at 0V, but should no longer result in error messages.

**Comment by marmitol** [comment]
I have installed the version beta v1.0.3-0?. But it does not work.

![image](https://user-images.githubusercontent.com/61157812/196144841-a2248323-3382-4ae3-812d-0129cb2bb192.png)
![image](https://user-images.githubusercontent.com/61157812/196144453-f3ccbedb-132a-4678-8672-145436a0fd99.png)
![image](https://user-images.githubusercontent.com/61157812/196145111-fd54a8f6-8ff5-43e9-9a4d-f6f659c11210.png)

Thank You

**Comment by ebaauw** [comment]
Could you please run Homebridge in DEBUG mode, and capture a bit more of the logs, in particular the debug line that lists the UPS state?  Like below:
```
[10/17/2022, 1:28:32 PM] [UPS] pi4: state: {"status":["OL"],"currentConsumption":20.8,"voltage":238,"mute":false,"testResult":"No test initiated","batteryLevel":100,"lowBatteryThreshold":10,"batteryVoltage":13.3,"batterRuntime":3168}
```
Thanks.

**Comment by marmitol** [comment]
Hello I attach the log file in mode debug.
[homebridge.log](https://github.com/ebaauw/homebridge-ups/files/9811259/homebridge.log)

**Comment by ebaauw** [comment]
Still a bug in getting the consumption.

Not quite sure where the warning about voltage comes from.  If it's still there, better delete the cached accessory.

Could you try beta v1.0.3-1?

**Comment by marmitol** [comment]
Hi. 
Yes, I have installed the indicated version “ beta v1.0.3-1” and it seems to be working fine. I attach a photo. If you need anything else you can ask me. Thank you very much.

**Comment by ebaauw** [comment]
Cool.  Is the 255W consumption (about) accurate?  What devices are you powering through the UPS?  How long have you been running Homebridge UPS?  It looks like maybe the total consumption is too high?  44 kWh / 255 W > 1 week?

**Comment by marmitol** [comment]
The truth is that I have several devices connected. A firewall, a router, two rassbery Berry pi with two Homebridge central units. And other home automation devices. That's why the consumption is high. I can have a battery autonomy of almost 45 minutes by lowering the consumption.

**Comment by ebaauw** [comment]
I think I report _Total Consumption_ in Wh, but HomeKit expects kWh.  So the value is a factor 1,000 off.  The history values are correct (Eve expects these in Wh).

**Comment by marmitol** [comment]
The total consumption shown is not correct. Consumption is mutilated with 1000.
![image](https://user-images.githubusercontent.com/61157812/198549436-0a661133-1e2a-45e7-8bae-2bdeb45ef7a3.png)

**Comment by ebaauw** [comment]
Published v1.0.3.

**Comment by marmitol** [comment]
Hello.
I have already updated to the latest version. But I was getting the wrong data in "Coste total" . I have proceeded to uninstall and reinstall to reset the counters to zero in "Coste total". If I don't say anything else, it has been solved. But I believe that with this version everything will work fine.
Thank you very much.

**Comment by ebaauw** [comment]
_Total Consumption_ was still wrong v1.0.3.  Fixed in v1.0.4 and unpublished 1.0.3.

**Comment by marmitol** [comment]
Ok

Thank
